src/LinearMaps.jl

```diff
@@ -11,6 +11,9 @@ using SparseArrays
 
 import Statistics: mean
 
+using ChainRulesCore
```

Comment:
```suggestion
using ChainRulesCore: unthunk, NoTangent, @thunk
```
It helps others know where to look for odd words like "thunk".  Up to you...